.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,33 @@
+name: github pages
+
+on:
+  push:
+    branches:
+      - main  # Set a branch to deploy
+  pull_request:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true  # Fetch Hugo themes (true OR recursive)
+          fetch-depth: 0    # Fetch all history for .GitInfo and .Lastmod
+
+      - name: Setup Hugo
+        uses: peaceiris/actions-hugo@v2
+        with:
+          hugo-version: 'latest'
+          # extended: true
```

Comment:
Would it be sufficient to remove this comment to get the extended version?

Reply:
Yes, done that in b5b81ab2035dd4649b58e24ae438b2634d8fbe02